operators/src/processing/time_shift.rs

```diff
@@ -268,12 +321,39 @@ where
         query: VectorQueryRectangle,
         ctx: &'a dyn QueryContext,
     ) -> Result<BoxStream<'a, Result<Self::VectorType>>> {
+        let time_start_difference = query.time_interval.start() - self.time_interval.start();
+        let time_end_difference = query.time_interval.end() - self.time_interval.end();
+
         let query = VectorQueryRectangle {
             spatial_bounds: query.spatial_bounds,
             time_interval: self.time_interval,
             spatial_resolution: query.spatial_resolution,
         };
-        self.processor.vector_query(query, ctx).await
+        let stream = self.processor.vector_query(query, ctx).await?;
+
+        let stream = stream.map(move |collection| {
+            let collection = collection?;
+
+            let time_intervals = collection
+                .time_intervals()
+                .iter()
+                .map(|time| {
+                    TimeInterval::new(
+                        time.start() + time_start_difference.num_milliseconds(),
```

Comment:
is this the correct point in time to snap to? I believe in VAT we snap everything to the start/end of the absolute shift.

Reply:
is this the correct point in time to set? I believe in VAT we set everything to the start/end of the absolute shift.